README.md

```diff
@@ -675,6 +684,28 @@ storageAccounts:
       - name: table02
 
 ```
+##### Example 6 : Create 1 storage account and configure properties for the Storage Account FileService
+
+```
+storageAccounts:
+  - name: storage01
+    storageAccountsFileService:
+      deleteRetentionPolicy:
+        enabled: true
+        days: 20
+```
+##### Example 7 : Create 2 file share, one with default properties other one with specific properties
+
+```
+storageAccounts:
+  - name: storage01
+    owner: "No"               --Note owner is set to 'No' to indicate storage account already exists and is owned by a different service in the team
```

Comment:
we able to get any dynamic settings here?

Reply:
the owner flag

Reply:
We'll revisit the owner implementation at later point.